src/wp-includes/Requests/library/Requests.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+/**
+ * This file includes the old Requests class
+ * for plugins or themes that still use it.
+ *
+ * @package WordPress
+ * @since 6.2.0
+ */
```

Comment:
```suggestion
<?php
/**
 * Placeholder file to load the Requests class when
 * a plugin or theme attempts to directly load the
 * `library/Requests.php` file.
 *
 * This file is deprecated, use `wp-includes/class-requests.php` instead.
 *
 * @deprecated 6.2.0
 * @package WordPress
 * @subpackage Requests
 * @since 6.2.0
 */
```

Reply:
That works for me.